Content.Server/Botany/Systems/BotanySystem.Seed.cs

```diff
@@ -140,18 +140,18 @@ public IEnumerable<EntityUid> Harvest(SeedData proto, EntityUid user, int yieldM
 
         var name = Loc.GetString(proto.DisplayName);
         _popupSystem.PopupCursor(Loc.GetString("botany-harvest-success-message", ("name", name)), user, PopupType.Medium);
-        return GenerateProduct(proto, Transform(user).Coordinates, yieldMod);
+        return GenerateProduct(proto, potencyBonus, Transform(user).Coordinates, yieldMod);
     }
 
-    public IEnumerable<EntityUid> GenerateProduct(SeedData proto, EntityCoordinates position, int yieldMod = 1)
+    public IEnumerable<EntityUid> GenerateProduct(SeedData proto, float potencyBonus, EntityCoordinates position, float yieldMod = 1f)
     {
         var totalYield = 0;
         if (proto.Yield > -1)
         {
-            if (yieldMod < 0)
-                totalYield = proto.Yield;
+            if (yieldMod < 0f)
+                totalYield = 0;
             else
-                totalYield = proto.Yield * yieldMod;
+                totalYield = (int) Math.Round((float) proto.Yield * yieldMod);
```

Comment:
Remove space after cast.

Reply:
This was not resolved.

Reply:
I though putting spaces after cast was the appropriate formatting.
My IDE complains when there aren't spaces.

Reply:
Your IDE is not authoritative with respect to style guidelines. If you are using a style configuration file that we provided, please file a bug reporting the inconsistency.

The [official C# documentation](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/types/casting-and-type-conversions) shows that there should be no space after the cast.

Reply:
Huh, really did not expect the IDE to be wrong like that.
Will fix those then.